test/extended/util/cluster/cluster.go

```diff
@@ -104,12 +104,6 @@ func DiscoverConfig() (*ClusterConfiguration, error) {
 		networkPluginIDs = append(networkPluginIDs, string(networkSpec.DefaultNetwork.Type)+"/"+string(networkSpec.DefaultNetwork.OpenShiftSDNConfig.Mode))
 	}
 
-	if p.Type == configv1.NonePlatformType {
```

Comment:
Also non controversial, could split with the other commits and get it merged faster